.github/workflows/build-spack.yml

```diff
@@ -96,8 +97,9 @@ jobs:
       ### ENVIRONMENT ###
 
       - name: Check out Spack
-        uses: spack/setup-spack@5792f7c7055c3707819380b5e3831d2be6e64b6c # 2024/12
+        uses: sethrj/setup-spack@200141be54665bd75af4dbb131d08d24ea606a9c
         with:
+          repository: celeritas-project/spack
```

Comment:
Is this temporary because Spack doesn't have g4vg@1.0.4 yet, or do we want to use our own fork of Spack. If so, maybe we should transfer the `sethrj/setup-spack` to Celeritas organization?

Reply:
I'm just waiting for https://github.com/spack/setup-spack/pull/19 to get merged in... hopefully won't take long!